src/OverlayTrigger.js

```diff
@@ -82,189 +81,175 @@ const defaultProps = {
 };
 
 // eslint-disable-next-line react/no-multi-comp
-class OverlayTrigger extends React.Component {
-  constructor(props, context) {
-    super(props, context);
-
-    this.trigger = React.createRef();
-    this.state = {
-      show: !!props.defaultShow,
-    };
-
-    // We add aria-describedby in the case where the overlay is a role="tooltip"
-    // for other cases describedby isn't appropriate (e.g. a popover with inputs) so we don't add it.
-    this.ariaModifier = {
-      enabled: true,
-      order: 900,
-      fn: data => {
-        const { popper } = data.instance;
-        const target = this.getTarget();
-        if (!this.state.show || !target) return data;
-
-        const role = popper.getAttribute('role') || '';
-        if (popper.id && role.toLowerCase() === 'tooltip') {
-          target.setAttribute('aria-describedby', popper.id);
-        }
-        return data;
-      },
+const OverlayTrigger = ({
```

Comment:
to match the other components, could we have this as:

```js
function OverlayTrigger({ ... })
```

i.e. we don't use arrow functions except with `React.forwardRef`